src/sentry/api/endpoints/setup_wizard.py

```diff
@@ -17,7 +17,7 @@
 SETUP_WIZARD_CACHE_TIMEOUT = 600
 
 
-@region_silo_endpoint
+@control_silo_endpoint
```

Comment:
This endpoint needed to be moved since it relies on reading from the cache that the other setup endpoint populates.